docs/2.10.0/docs/canton/usermanual/troubleshooting_guide.rst

```diff
@@ -325,6 +325,79 @@ Common patterns from the canton log:
 
           The log line contains the name of the missing claim, but not the actual claims. When using tokens based on user names (`Audience Based Tokens <https://docs.daml.com/app-dev/authorization.html#audience-based-tokens>`_ and `Scope Based Tokens <https://docs.daml.com/app-dev/authorization.html#scope-based-tokens>`_), consult the user management service to see whether you need to grant more rights to the user. When using tokens based on party names (`Custom Claims Access Tokens <https://docs.daml.com/app-dev/authorization.html#custom-daml-claims-access-tokens>`_), debug the token in `JWT.IO <https://jwt.io/#debugger-io>`_.
 
+Disconnections
+--------------
+
+Intermittent network failures cause the majority of disconnections that occur in real-life scenarios. Less frequently,
+disconnections happen when network proxies, load balancers, or other elements of the network infrastructure terminate the client-server
+connection because they think that there is no activity going on in the communication channel. Such disconnections are predictable and
+can be avoided by carefully configuring the components' keep-alive parameters.
```

Comment:
Would recommend to be a bit careful about how the scope of the problem is represented here, as load balancers can contribute to disconnections for many and varied reasons.

For example, I worked on an issue for HKEX where a ledger client could not consume the output of `GetActiveContracts` fast enough, so backpressure from the client onto the load balancer triggered disconnection once nginx's buffer (of configurable size) was over-filled.